main.go

```diff
@@ -98,7 +103,11 @@ func glob(sources []string) ([]string, error) {
 		if err != nil {
 			return nil, err
 		}
-		paths = append(paths, matches...)
+		if matches == nil && !strings.ContainsAny(source, "*?[]^") {
```

Comment:
Does this deserve a test for itself ?

Reply:
Yes, also thinking about it I believe we should only support `*` and `?`…